src/DssSpell.sol

```diff
@@ -19,76 +19,67 @@ pragma solidity 0.8.16;
     function ilks(bytes32 ilk) external view returns (uint256 Art, uint256 rate, uint256 spot, uint256 line, uint256 dust);
 }
 
+interface DssVestLike {
+    function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
+    function file(bytes32, uint256) external;
+    function restrict(uint256) external;
+}
 interface RwaLiquidationLike {
-    function ilks(bytes32) external view returns (string memory doc, address pip, uint48 tau, uint48 toc);
-    function init(bytes32 ilk, uint256 val, string calldata doc, uint48 tau) external;
+    function ilks(bytes32) external view returns (string memory, address, uint48, uint48);
+    function init(bytes32, uint256, string calldata, uint48) external;
+}
+
+interface ACLManagerLike {
+    function addPoolAdmin(address admin) external;
+}
+
+interface ProxyLike {
+    function exec(address target, bytes calldata args) external payable returns (bytes memory out);
+}
+
+interface Initializable {
+    function init(bytes32 ilk) external;
 }
 
 interface RwaUrnLike {
-    function lock(uint256 wad) external;
     function hope(address usr) external;
+    function nope(address usr) external;
+    function lock(uint256 wad) external;
+    function draw(uint256 wad) external;
 }
 
 interface RwaInputConduitLike {
     function mate(address usr) external;
+    function hate(address usr) external;
     function file(bytes32 what, address data) external;
 }
 
 interface RwaOutputConduitLike {
     function file(bytes32 what, address data) external;
     function hope(address usr) external;
+    function nope(address usr) external;
     function mate(address usr) external;
+    function hate(address usr) external;
     function kiss(address who) external;
-}
-
-interface PoolConfiguratorLike {
-    struct InitReserveInput {
-        address aTokenImpl;
-        address stableDebtTokenImpl;
-        address variableDebtTokenImpl;
-        uint8 underlyingAssetDecimals;
-        address interestRateStrategyAddress;
-        address underlyingAsset;
-        address treasury;
-        address incentivesController;
-        string aTokenName;
-        string aTokenSymbol;
-        string variableDebtTokenName;
-        string variableDebtTokenSymbol;
-        string stableDebtTokenName;
-        string stableDebtTokenSymbol;
-        bytes params;
-    }
-    function initReserves(InitReserveInput[] calldata input) external;
-    function configureReserveAsCollateral(
-        address asset,
-        uint256 ltv,
-        uint256 liquidationThreshold,
-        uint256 liquidationBonus
-    ) external;
-    function setBorrowableInIsolation(address asset, bool borrowable) external;
-    function setDebtCeiling(address asset, uint256 newDebtCeiling) external;
-}
-
-interface AaveOracleLike {
-    function setAssetSources(address[] calldata assets, address[] calldata sources) external;
+    function pick(address who) external;
+    function push() external;
```

Comment:
By that metric we shouldn't remove anything unused in the current spell because it makes porting to mainnet easier.

I'm OK to comment it out, just don't leave it in there and active I guess?